shared/src/business/entities/EntityConstants.ts

```diff
@@ -634,29 +636,39 @@ export const INITIAL_DOCUMENT_TYPES = {
     documentTitle: 'Application for Waiver of Filing Fee',
     documentType: 'Application for Waiver of Filing Fee',
     eventCode: 'APW',
+    tabTitle: 'APW',
+    sort: 5,
```

Comment:
I'm seeing the same sort(b - a) thing in a lot of different places because of this change. is there a better way to accomplish it?